cli/config.go

```diff
@@ -423,6 +439,12 @@ func updateRuntimeConfig(configPath string, tomlConf tomlConfig, config *oci.Run
 		config.ShimConfig = shConfig
 	}
 
+	if config.HypervisorConfig.UseVSOCK {
```

Comment:
Could you add one comment here saying that having the Vsock enabled will not take the proxy config suggested by the configuration file, and will simply override it with the NoProxy configuration.
Just to make sure this is clear that we're overriding what has been asked through the config.